.travis.yml

```diff
@@ -17,6 +17,8 @@ notifications:
 install:
   - pip install -U pip setuptools
   - pip install -U coveralls coverage
+  - pip install -e  git+https://github.com/zopefoundation/zope.app.basicskin.git@python3#egg=zope.app.basicskin
+  - pip install -e  git+https://github.com/zopefoundation/zope.app.pagetemplate.git@python3#egg=zope.app.pagetemplate
```

Comment:
These will be removed once those two packages are fixed and released, yes?

(This is not really a question, more like a reminder.)